### ISSUE TITLE
Some microsite image metadata is not being imported correctly

### ISSUE THREAD

**Issue by eddierubeiz** [sciencehistory/scihist_digicoll#1108]
Example: OH `0083B`
No alt text or caption for photo in staging

Source:
    https://oh.sciencehistory.org/oral-histories/hackerman-norman-1
       
Destination:
    https://staging-digital.sciencehistory.org/admin/works/tb09j665k#nav-metadata
    https://staging-digital.sciencehistory.org/admin/asset_files/1wg46h5
Headshot does not have alt text or caption
